mir-ci/mir_ci/test_tests.py

```diff
@@ -57,3 +63,26 @@ async def test_program_is_killed_when_terminate_fails(self) -> None:
         elapsed = time.time() - start
         assert p.output.strip() == 'abc\nijk'
         assert abs(elapsed - 2.5) < 0.1
+
+    def test_program_command_has_prefix_when_systemd_slice_is_set(self) -> None:
+        p = Program(['sh', '-c', 'sleep 1; echo abc'], systemd_slice="test-slice")
+        assert p.command == ("systemd-run", "--user", "--scope", "--slice=test-slice", "sh", "-c", "sleep 1; echo abc")
+
+
+class TestBenchmarker:
```

Comment:
This marks the test as a "self" test, which lets us only run these here in CI, rather than when these tests run on new Mir changes:
```suggestion
@pytest.mark.self
```

Also coverage is only measured based on self tests.